gio/ginetaddress.c

```diff
@@ -521,9 +540,15 @@ g_inet_address_new_from_string (const gchar *string)
   g_networking_init ();
 
   if (inet_pton (AF_INET, string, &in_addr) > 0)
+  {
     return g_inet_address_new_from_bytes ((guint8 *)&in_addr, AF_INET);
+  }
```

Comment:
Let's not add this block, want to keep changes on top of upstream code to a minimum.

Reply:
Sure. This one and the next weren't needed. I checked through the rest and they all incorporated the following structure:

else
{
#ifdef HAVE_IPV6
  do something
#else
  do something else
#endif
}

These technically could be reduced to a single line if, as the pre-processor would reduce the code before compilation, however although the 'contributing' page didn't explicitly say what I should do, I thought that this may make the code hard to read so I introduced a braced block where a single-line else clause had been used before.

Subsequently, the 'contributing' page said to use a braced block for both parts of the if/else if either uses them and so I introduced braces on the if clause too match. Again, I am not sure whether you agree this is the right approach as it isn't so clear cut when you are working with third-party code how much you want to retrofit your own coding style.

Happy to change things either way, just let me know what works best.